src/components/TodoList/TodoList.tsx

```diff
@@ -1,4 +1,19 @@
-// Add the required types and props
-export const TodoList = () => (
-  <>TodoList markup</>
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  todos: Todo[]
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => (
+  <ul>
+    <section className="TodoList">
+      {todos.map(todo => (
+        <li>
+          <TodoInfo key={todo.id} todo={todo} />
+        </li>
```

Comment:
Not sure about this lis, because original design didn't have them

Reply:
better add the key to a topmost element in array
```suggestion
        <li key={todo.id}>
          <TodoInfo todo={todo} />
        </li>
```